packages/react/toast/src/Toast.tsx

```diff
@@ -680,18 +686,21 @@ ToastClose.displayName = CLOSE_NAME;
 
 /* ---------------------------------------------------------------------------------------------- */
 
-function handleAndDispatchDiscreteCustomEvent<
-  E extends CustomEvent,
-  ReactEvent extends React.SyntheticEvent
->(
+function dispatchCustomEvent<E extends CustomEvent, ReactEvent extends React.SyntheticEvent>(
```

Comment:
Did you rename because of the wrapping?
Didn't we have a similar function in another component? I'd rather keep consistent if they essentially do the same thing (minus the discrete boolean caveat)

Reply:
> Didn't we have a similar function in another component

I kept the naming here due to the additional arg vs the other case which always dispatches `discrete`.

Perhaps they should align them both to be `handleAndDispatchCustomEvent`?

Reply:
Yeah I think so, as that's what they're both doing.